morph/react/block-animated.js

```diff
@@ -0,0 +1,30 @@
+import { getTimingScopes } from '../utils.js'
+
+export function enter(node, parent, state) {
+  if (!node.hasTimingAnimation) return
+
+  state.render.push(
```

Comment:
Shouldn't we only render this if the `onAnimationDone` slot exists and skip it otherwise @amymc?

Reply:
Hey @dariocravero, maybe I've misunderstood how this should be implemented.
According to [the original issue](https://github.com/viewstools/morph/issues/111#issue-338669266) there is no slot for `onAnimationDone`, the view file is unchanged. `onAnimationDone` is passed in as a prop from the logic file.